src/dynamic/dynamic-attributes.directive.spec.ts

```diff
@@ -42,7 +42,7 @@ describe('DynamicAttributesDirective', () => {
           TestComponent,
           ComponentOutletInjectorDirective,
         ],
-        providers: [{ provide: COMPONENT_INJECTOR, useValue: null }],
+        providers: [{ provide: COMPONENT_INJECTOR, useValue: DynamicComponent }],
```

Comment:
@kamilkisiela hmm, well this token itself is not important because `ngComponentOutlet` is playing the role of `COMPONENT_INJECTOR` so I set it to null.

It seems something changed in angular and now null is breaking...

But I guess we can leave it like that since the tests are passing again =)